### PR TITLE
Calculate Dashboard Current Liquid Assets

### PR DESCRIPTION
**Pull request by josemontesdeoca**
quick and dirty liquid assets calculation for prototype purposes. 

@fcduarte @amitbharadwaj please review.

### CODE REVIEW

File: src/org/missionassetfund/apps/android/fragments/DashboardFragment.java [hunk 1]
```diff
@@ -72,11 +79,47 @@ private void setupViews(View v) {
...
+                    Double cla = 0d;
+                    Double spentToday = 0d;
+                    Date startOfToday = MAFDateUtils.getStartOfToday();
+                    for (Transaction t : results) {
+                        if (t.getType().equals(TransactionType.CREDIT)) {
+                            cla += t.getAmount();
+                        } else {
+                            cla -= t.getAmount();
+                            // Check Spents Today.
+                            if (t.getTransactionDate().after(startOfToday)) {
```
Comment (fcduarte) [inline src/org/missionassetfund/apps/android/fragments/DashboardFragment.java]: http://developer.android.com/reference/android/text/format/DateUtils.html#isToday(long)

Comment (josemontesdeoca) [inline src/org/missionassetfund/apps/android/fragments/DashboardFragment.java]: ah! let me use that instead! :+1:

**Comment by josemontesdeoca** [comment]
@fcduarte removed the added function and code for getting current day and used DateUtils.isToday() method! :+1:

**Comment by fcduarte** [comment]
Reviewed.

**Pull request merged**